resources.md

```diff
@@ -0,0 +1,132 @@
+# Resources
+
+This page contains informations about various tools and technologies
+that are useful to anyone developing on Knative.
+
+## Community Resources
+
+This section contains tools and technologies developed by members of the
+Knative community specifically for use with Knative.
+
+### [`knctl`](https://github.com/cppforlife/knctl)
+
+`knctl` is an under-development command line tool for working with
```

Comment:
Link to more information? Otherwise if there isn't any more than this I think we should remove it until there is more info

Reply:
There is a link to the knctl repo, and there's some documentation there about how to work with it. What other information would you want included here? I could add that it's an alternative to working directly with kubectl.

Reply:
Each of the other sections has at least a couple explanatory/introductory paragraphs, even though you could link to each repo and just tell people to read there.

It's not a big deal, it just feels light compared to the others.